### PR TITLE
docs: describe how to disable usage stats with kuberay

### PR DESCRIPTION
**Pull request by davidxia**
Signed-off-by: David Xia <david@davidxia.com>

### CODE REVIEW

**Review by DmitriGekhtman — APPROVED** [review]

**Review by jjyao — APPROVED** [review]

File: doc/source/cluster/usage-stats.rst [hunk 1]
```diff
@@ -37,6 +37,8 @@ There are multiple ways to disable usage stats collection before starting a clus
 
 #. Set the environment variable ``RAY_USAGE_STATS_ENABLED`` to 0 (e.g., ``RAY_USAGE_STATS_ENABLED=0 ray start --head`` :ref:`command <ray-start-doc>`).
 
+#. If you're using `kuberay <https://ray-project.github.io/kuberay/>`_, you can add ``disable-usage-stats: 'true'`` to ``.spec.[headGroupSpec|workerGroupSpecs].rayStartParams.``.
```
Comment (architkulkarni) [inline doc/source/cluster/usage-stats.rst:40]:
```suggestion
#. If you're using `KubeRay <https://ray-project.github.io/kuberay/>`_, you can add ``disable-usage-stats: 'true'`` to ``.spec.[headGroupSpec|workerGroupSpecs].rayStartParams.``.
```

**Comment by davidxia** [comment]
Hi need another review and merge on this. @architkulkarni

**Comment by clarng** [comment]
This merge did not have @zhe-thoughts approval, as required by the branch freeze.

cc @zhe-thoughts on approval or revert

**Comment by zhe-thoughts** [comment]
Retrospective approval

**Pull request merged**